Gemfile.lock

```diff
@@ -242,6 +266,7 @@ DEPENDENCIES
   factory_girl_rails (~> 4.0)
   figaro
   jquery-rails
+  mailcatcher
   omniauth-facebook
   omniauth-google-oauth2
   pg
```

Comment:
Mailcatcher is necessary for testing emails. I'm not entirely sure about the rest of the gems here, though. Didn't add them.